test/manual/kerberos.test.js

```diff
@@ -97,6 +97,42 @@ describe('Kerberos', function () {
     });
   });
 
+  context('when passing SERVICE_HOST as an auth mech option', function () {
+    context('when the SERVICE_HOST is invalid', function () {
+      const client = new MongoClient(`${krb5Uri}&maxPoolSize=1`, {
+        authMechanismProperties: {
+          SERVICE_HOST: 'example.com'
+        }
+      });
+
+      it('fails to authenticate', async function () {
+        let expectedError;
+        await client.connect().catch(e => {
+          expectedError = e;
+        });
+        if (!expectedError) {
+          expect.fail('Expected connect with invalid SERVICE_HOST to fail');
+        }
+        expect(expectedError).property('message').to.include('Authentication failed');
```

Comment:
This test failed with a different error in CI - does it fail with this error locally?

Reply:
Oh I had my local env setup wrong. I've updated now to check regex as the macos and linux error messages are slightly different.